engines/python/setup/djl_python_engine.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, args, service):
 
             self.clean_up()
         elif self.sock_type == "tcp":
-            self.sock_name = "127.0.0.1"
+            self.sock_name = socket.gethostname()
```

Comment:
We should try to use `sock_name` from frontend

Reply:
But socket name 0.0.0.0 doesn't work.

Reply:
Changed to 0.0.0.0